MediaBrowser.Model/Entities/MediaStream.cs

```diff
@@ -220,6 +234,7 @@ private string GetResolutionText()
             return null;
         }
 
+        /*
         private string AddLanguageIfNeeded(string title)
```

Comment:
can you remove dead code instead of commenting it? makes reading reviews harder...

Reply:
Sure, sorry for that.